### ISSUE TITLE
Activity inflated 14 times?

### ISSUE THREAD

**Issue by kjeremy** [johnthiriet/MvvmCrossBindingInflatePerformanceTest#2]
Your readme says that the Activity inflates 14 times.  Do you mean that you RAN this program 14 times?  It doesn't appear to inflate the activity more than once.

**Comment by johnthiriet** [comment]
That's correct. That's all I could do during the 2 hours my wife slept this sunday afternoon. That's also why I said it was the first results and it would need more investigation.
